docs/developer-guide/etendo-classic/how-to-guides/how-to-create-client-event-handler-actions.md

```diff
@@ -0,0 +1,125 @@
+---
+tags:
+  - client event handler
+  - actions
+  - Etendo Classic
+---
+
+#  How to Create Client Event Handler Actions
+  
+##  Overview
+
+  This section discusses how to implement client-side functions (JavaScript) that are executed before or after an event is fired in a standard window of the User Interface.
+  
+##  Example Module
+
+  This is supported by an example module which shows examples of the code shown and discussed.
+
+  The code of this module can be downloaded from [this repository](https://github.com/etendosoftware/com.etendoerp.client.application.examples/blob/main/src/com/etendoerp/client/application/examples/GreetingEventHandler.java).
+
+##  Defining Client Event Handler Actions
+
+  A **Client Event Handler is a component that allows developers to respond to specific events in Etendo**, such as the creation, update or deletion of entities. These handlers are essential for implementing custom business logic that executes when certain changes occur in the database.
+
+### Step 1: Creating an Action
+
+  * Navigate to the **Process Definition** window.
+  * Create a new record and define the **UI pattern** as **Action**.
+  * Mark the process as **Multi Record**.
+
+!!!info
+    Check the box to indicate that the process is **Multi Record** (if it is necessary to handle multiple registrations at the same time).
+
+![createjobs1.png](https://docs.etendo.software/latest/assets/legacy/technicaldocumentation/platform/createjobs1.png)
+
+### Step 2: Create a Button to Execute the Action
```

Comment:
What's the purpose of this step?
I mean, if it's an Event Handler to listen events for a specific entity, why a button is needed?

Reply:
Is it okay to allude that an example would be to create a button?
'An example, creating a button to execute the action'

Reply:
Could be, yes, but if you need just something that trigger an update or a save, maybe it's better to make a simple change to a row.
Maybe @isaiasb-etendo could enlighten us with this :stuck_out_tongue_closed_eyes:

Reply:
@RomanMagnoli , según veo en la [Documentación de OB](https://wiki.openbravo.com/wiki/How_to_create_client_event_handler_actions) había funciones de pre y post save que según lo que me comento Iván en Etendo esto cambio... De ser así, yo sugiero que intentemos seguir el ejemplo pero de la nueva forma. ¿Se pueden ejecutar funciones equivalentes?

Reply:
@RomanMagnoli Don't know if you see this comment. I @ you just in case.

Reply:
Sorry... Answering Isa, I don't know if there are any equivalent function, but If you took it in mind it ok for me!

Reply:
@isaiasb-etendo  @RomanMagnoli Hello, I wanted to consult if the changes made are correct to improve the understanding about the “Event Handler”. I have removed the section that created a button, as it was not directly related to the topic.